src/components/container/Octagon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import '@/styles/shapes.css';
+
+interface OctagonProps {
```

Comment:
would change this and Hexagon props to the same style as the other components :)